ray-operator/Makefile

```diff
@@ -4,7 +4,8 @@ COMMIT_SHA1     := $(shell git rev-parse HEAD )
 # Image URL to use all building/pushing image targets
 IMG ?= controller:latest
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:maxDescLen=100,trivialVersions=true,preserveUnknownFields=false,generateEmbeddedObjectMeta=true"
+# Golang forbids converting float from Json to struct. Need allowDangerousTypes=true to unblock build.
+CRD_OPTIONS ?= "crd:maxDescLen=100,trivialVersions=true,preserveUnknownFields=false,generateEmbeddedObjectMeta=true,allowDangerousTypes=true"
```

Comment:
Floats are considered bad due to varying standards and the lack of proper json support.
Is there a way to avoid floats? I haven't looked carefully, but if it's a timeout, will seconds work?

Reply:
I see it's used for time-related things in units of seconds.
Would int be sufficient granularity?

Reply:
I think for time, we can use int. The things required for float are numCpus and numGpus

Reply:
As discussed offline, fuzziness around resource quantities is inherent to Ray APIs, so we can't meaningfully get around floats there.